systers_portal/templates/meetup/meetup.html

```diff
@@ -3,10 +3,16 @@
 {% block meetup_location_page_content %}
   <h2 id="meetup-title">{{ meetup.title }}</h2>
   <div class="box-container box-body pt25">
-    <p> 
+    <p>
       <b> Date: </b> {{ meetup.date }}
-      &nbsp; &nbsp; 
+      {% if meetup.end_date %}
+      <b> - </b> {{ meetup.end_date }}
+      {% endif %}
+      &nbsp; &nbsp;
```

Comment:
have you checked whether the responsiveness is getting affected by this or not ?

Reply:
@abhi20161997 Responsiveness is not affected works fine for all devices.